common/djangoapps/tedix_ro/forms.py

```diff
@@ -162,7 +163,7 @@ def clean_parent_phone(self):
         user = User.objects.filter(email=parent_email).first() if parent_email else None
         # TODO: there is a validator field, remove the additional validation of
         # the length of the phone number in the future
-        if len(parent_phone) < 10 or len(parent_phone) > 15:
+        if len(parent_phone) != 0 and len(parent_phone) < 10 or len(parent_phone) > 15:
```

Comment:
is this change really needed? It seems to me that it's the same case as it was before...

Reply:
for an optional field, this condition is needed, if we leave an empty field and click on Create Account, we will get a ValidationError(_('The parent phone number length must be from 10 to 15 digits.')

Reply:
ok, then can it be like this `parent_phone and len(parent_phone) < 10 or len(parent_phone) > 15`?